packages/aws-cdk-lib/aws-stepfunctions-tasks/test/evaluate-expression.test.ts

```diff
@@ -89,17 +89,17 @@ test('with dash and underscore in path', () => {
   });
 });
 
-test('With Node.js 18.x', () => {
+test('With Node.js 20.x', () => {
```

Comment:
Generally we don't want to change any existing tests. If they were accepted at one point they should continue working event with future changes otherwise it signal that we're making a breaking change. I would keep the existing node18 test and add a new test for node20. In fact, maybe we just want to keep an array of runtime versions and use `forEach` to test each case. As an example - https://github.com/aws/aws-cdk/blob/f4b825e34f08d29bea19fe29be422fe6143acabd/packages/%40aws-cdk/aws-glue-alpha/test/job-executable.test.ts#L132

Reply:
Thanks @colifran ,I updated in lambda-node-js too, let me fix in both cases

Reply:
EvalNodejsSingletonFunction has now runtime as nodejs20 updated in generated provider through framework:
aws-cdk/packages/aws-cdk-lib/custom-resource-handlers/dist/aws-stepfunctions-tasks/eval-nodejs-provider.generated.ts

` "runtime": lambda.Runtime.NODEJS_20_X`

 even though below function is taking the runtime as input props, only the uuid is being passed to provider, runtime remains `nodejs20`, so test case for Node.js 18x seems to be no longer valid , let me know if my understanding is not correct
```
function createEvalFn(runtime: lambda.Runtime | undefined, scope: Construct) 

.......
return new EvalNodejsSingletonFunction(scope, 'EvalFunction', {
    uuid,
    lambdaPurpose,
  });
```

```
export interface EvalNodejsSingletonFunctionProps extends lambda.FunctionOptions {
  readonly uuid: string;
  readonly lambdaPurpose?: string;
}
```